src/view/pullRequestCommentController.ts

```diff
@@ -389,19 +390,15 @@ export class PullRequestCommentController implements CommentHandler, CommentReac
 		}
 	}
 
-	public async finishReview(thread: GHPRCommentThread, input: string): Promise<void> {
-		try {
-			await this.createOrReplyComment(thread, input, false, false);
-			await this.pullRequestModel.submitReview();
-			this.setContextKey(false);
-		} catch (e) {
-			vscode.window.showErrorMessage(`Failed to submit the review: ${e}`);
-		}
-	}
 
-	public async deleteReview(): Promise<void> {
-		await this.pullRequestModel.deleteReview();
-		this.setContextKey(false);
+	public async openReview(): Promise<void> {
+		await PullRequestOverviewPanel.createOrShow(this._folderReposManager.context.extensionUri, this._folderReposManager, this.pullRequestModel);
+		PullRequestOverviewPanel.openReview();
```

Comment:
Why is a call to `openReview` needed? Does `createOrShow` not actually show?

Reply:
Renamed to `scrollToReview` for clarity